server/src/test/java/io/pravega/schemaregistry/server/rest/resources/SchemaRegistryResourceTest.java

```diff
@@ -1,25 +1,46 @@
 /**
- * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Copyright (c) Dell Inc.,or its subsidiaries. All Rights Reserved.
+ * <p>
```

Comment:
dont change the license please

Reply:
Looks like the license changed. Have updated it in the all the files I have written.